### ISSUE TITLE
Add travis check for file headers

### ISSUE THREAD

**Issue by rauhul** [HackIllinois/iOS#123]
We should add a ci script to check that all swift files begin with a header in the format below. The script should live under `.ci/`.

```
//
//  <FILE_NAME>.swift
//  HackIllinois
//
//  Created by <FIRST> <LAST> on ##/##/##.
//  Copyright © <YEAR> HackIllinois. All rights reserved.
//  This file is part of the Hackillinois iOS App.
//  The Hackillinois iOS App is open source software, released under the University of
//  Illinois/NCSA Open Source License. You should have received a copy of
//  this license in a file with the distribution.
//

```

**Comment by rauhul** [comment]
I think this could be a custom `file_header` rule in SwiftLint

**Comment by sujaypat** [comment]
thanks

**Comment by sujaypat** [comment]
Resolved by #138
